lumicks/pylake/detail/bead_cropping.py

```diff
@@ -88,14 +105,178 @@ def find_beads(
         np.flip(only_beads[: bead_edges[1]]) < np.flip(blurred[: bead_edges[1]])
     )[0][0]
 
-    # We then keep moving until we've hit a low enough signal level that we are confident we
-    # have reached the tether.
-    final_cutoff = np.percentile(data[bead_edges[0] : bead_edges[1]], threshold_percentile)
-    bead_edges[0] += np.where(data[bead_edges[0] :] <= final_cutoff)[0][0]
-    bead_edges[1] -= np.where(np.flip(data[: bead_edges[1]]) <= final_cutoff)[0][0]
+    bead_edges = _move_inward_percentile(data, bead_edges, threshold_percentile)
 
     if plot:
         [plt.axvline(be, label="edge", color="k", linestyle="--") for be in bead_edges]
         plt.legend()
 
     return bead_edges
+
+
+def find_beads_template(
+    kymograph_image,
+    bead_diameter_pixels,
+    num_time_frames=5,
+    plot=False,
+    allow_movement=False,
+    threshold_percentile=70,
+):
+    """Determine bead edges using a cross correlation procedure
+
+    This function seeks a bead-sized template that appears in subsequent frames.
+    It then moves inward by half the template size to provide bead edges.
+
+    Parameters
+    ----------
+    kymograph_image : np.ndarray
+        2D kymograph image
```

Comment:
Yeah, I'll add a commit for making this consistent. I agree. I just held off on it because I wanted to constrain this PR to the new functionality, but it's good to just do it now.